src/app.service.ts

```diff
@@ -1,8 +1,4 @@
 import { Injectable } from '@nestjs/common';
 
 @Injectable()
-export class AppService {
-  getHello(): string {
-    return 'Hello World!';
-  }
-}
+export class AppService {}
```

Comment:
jeśli serwis jest nieużywany to trzeba go wywalić, nie ma co chomikować kodu na później